### DISCUSSION TITLE
I want to learn to hack, and I mean for real!

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#23088] [Q&A] [answered]
Can someone send me a link of some course for hacking?

**Comment by icecream17** [comment, marked as answer]
http://www.catb.org/~esr/faqs/hacker-howto.html

**Reply by Rhysjosmin** [reply]
https://www.youtube.com/channel/UC9x0AN7BWHpCDHSm9NiJFJQ

**Reply by ghost** [reply]
Thank you

**Comment by the-j0k3r** [comment]
> Can someone send me a link of some course for hacking?

The most misused word in the world and the word with the most negative connotation I know of in this topic.HACKER, HACKING, HACKED, all made infamous by opportunists, and thus your question makes me wonder.

Hacking in the right sense of the word is not what you envisaged when you started this topic, and because it makes you sound like you would like to become a script kiddie to get to non productive activities like disrupting networks and so on, then asking for a how to is really not going to get you the answers you want. So therefore I give none.
